velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -1617,6 +1617,20 @@ bool AggregationFuzzer::verifySortedAggregation(
              .planNode(),
          splits});
 
+    if (!groupingKeys.empty()) {
+      plans.push_back(
+          {PlanBuilder()
+               .tableScan(inputRowType)
```

Comment:
Let's also add a simple plan, but with values node instead of tableScan.

Reply:
Got it, added. 
I've debugged and run both Velox and Spark aggregation fuzzer. They all reach the breakpoint in the places that I added the streaming aggregation plan, and all success.
```
velox_aggregation_fuzzer_test --seed 0 --duration_sec 300 --enable_sorted_aggregations --logtostderr=1 --minloglevel=0 --repro_persist_path=./tmp/aggregate_fuzzer_repro

spark_aggregation_fuzzer_test --seed 0 --duration_sec 300 --enable_sorted_aggregations --logtostderr=1 --minloglevel=0 --repro_persist_path=./tmp/spark_aggregate_fuzzer_repro
```

Reply:
This is great. Would you also run these for longer. 1h each. Just to be absolutely sure.

Reply:
Also, please, remove --seed 0.

Reply:
Got it, will run it nightly :)